Doc/pages/gui.rst

```diff
@@ -1,16 +1,333 @@
 
-Using MDANSE Graphical User Interface
+.. _section-7:
+
+Section 7: Navigating the MDANSE GUI
```

Comment:
Definitely don't put any numbers in the title of a section. It's up to the documentation engine to assign numbers. Now if you move this text segment, the number will be wrong.